app/src/main/java/com/tip/lunchbox/data/zomato/ZomatoApiManager.java

```diff
@@ -27,11 +29,11 @@ private static Retrofit getRetrofitClient() {
     private static OkHttpClient getOkHttpClient() {
         HttpLoggingInterceptor logging = new HttpLoggingInterceptor();
         logging.setLevel(HttpLoggingInterceptor.Level.BODY);
-
         return new OkHttpClient()
                 .newBuilder()
                 .addInterceptor(new ZomatoApiInterceptor())
                 .addInterceptor(logging)
+                .readTimeout(5, TimeUnit.SECONDS)
```

Comment:
@epicadk we are not expecting a huge load on the server so I believe we can keep lower socket time out values. But still, I believe the connection time out value can be set as low as 5 seconds but the socket timeout can be dragged a little higher like 10 seconds.

What was your reasoning behind setting this as 5 seconds?